src/actions/blockchain.ts

```diff
@@ -268,11 +349,11 @@ export const approveAndPostSellOrder = (choice: string) => async (dispatch: Func
           body: `You are approving the maximum amount - you will no longer need to sign 2 transactions.`,
         },
       }))
-      
-      const tokenApprovalReceipt = await tokenApproval(sell, (100000 * 10 ** 18).toString())
+      const allowanceLeft = (await getTokenAllowance(sell, currentAccount)).toNumber()
+      const tokenApprovalReceipt = await tokenApproval(sell, ((2 ** 255) - allowanceLeft).toString())
```

Comment:
why `2 * 255`? Just because?

Reply:
2 ** 256 is highest number solidity accepts but I was having errors with 256 as I think some pre existing approvals were breaking the limit, something to look into later for now